Projects/TEAM-2/Klopotenko/224/224/Program.cs

```diff
@@ -0,0 +1,78 @@
+﻿/* Дано масив випадкових чисел в діапазоні від -20 до +20. Необхідно знайти 
+ позиції крайніх негативних елементів 
+ (самого лівого негативного елементу і самого правого негативного елементу)
+ і впорядкувати елементи, що знаходяться між ними. */
+
+
+
+using System;
+
+namespace _224
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int[] array = new int[10];
+            Console.WriteLine("Complete the array with 10 integer values from -20 to 20. Entrr \"Stop\" when you want finish: \n");
```

Comment:
"Stop" doesn't work...

Reply:
I removed "Stop" from project

Reply:
I understood this task that I need to find the smallest and the biggest negative numbers and sort all numbers from array between them. I will try to remake this task

Reply:
Keep it. I'm not sure that I'm right about the task.

Reply:
Ok